source/getting-started/use-cases/configuration-assessment.rst

```diff
@@ -81,43 +81,35 @@ The Wazuh dashboard has a **Configuration Assessment** module that allows you to
    :align: center
    :width: 80%
     
-In the image below, you see the policy based on the CIS benchmark for Ubuntu Linux 22.04.
-
-.. thumbnail:: /images/getting-started/use-cases/sca/cis-benchmark-ubuntu22-policy.png
-   :title: Policy for CIS benchmark for Ubuntu 22.04
-   :alt: Policy for CIS benchmark for Ubuntu 22.04
-   :align: center
-   :width: 80%
```

Comment:
**question**: why this image was removed?

Reply:
Because currently, the policy overview and results are in the same page.  The following image contains what this used to